tests/test_serialization.py

```diff
@@ -253,6 +253,9 @@ class BoolContainer:
     [
         ("2022-02-20T11:33:48.607289+00:00", datetime.datetime(2022, 2, 20, 11, 33, 48, 607289, datetime.timezone.utc)),
         ("2022-02-20T11:33:48.607289", datetime.datetime(2022, 2, 20, 11, 33, 48, 607289, datetime.timezone.utc)),
+        ("2022-02-20T11:33:48", datetime.datetime(2022, 2, 20, 11, 33, 48, 0, datetime.timezone.utc)),
```

Comment:
I believe, TZ absence means `tzinfo=None`

Reply:
The tests show that your observation is not correct.

Reply:
So, test is erroneus
```
> datetime.datetime.fromisoformat("2022-02-20T11:33:48").tzinfo is None
True
```

Reply:
We decided at the very beginning that datetimes here will always have a timezone. We serialize any datetime as if it has timezone (if it doesn't we assume that it is utc). We deserialize datetimes with timezone event if they don't have one in input (we assume utc for this cases, the same as for serialization). We strongly believe that passing datetimes without timezone through network is a bad idea.

Reply:
Let's discuss it separately, via issue or proposal from @timuram, because there is a slightly different behaviour in v2 of marshmallow and marshmallow[reco]